### ISSUE TITLE
Allow user to remove account if unable to verify email

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla-mobile/firefox-ios#5679]
If a user enters in email email they are unable or unwilling to verify, there is currently no mechanism in iOS to remove it.
![IMG_1274](https://user-images.githubusercontent.com/68213/67716343-01b26f00-f9a2-11e9-9f17-050047329eb9.PNG)

Consider adding a Remove button to the top right of the FxA web view.
![remove](https://user-images.githubusercontent.com/68213/67716901-0b88a200-f9a3-11e9-8947-ec64bdccd5c9.jpg)

**Comment by athomasmoz** [comment]
Hey @ryanfeeley how did you get to this state? I tried logging in, and when the email confirmation came up, I clicked done. Then when I re-opened that screen, I was back to the sign in page

**Comment by ryanfeeley** [comment]
@athomasmoz iOS has two ways to open the FxA web view. One is from the Settings (which I think is shown there) and the other is from all other sync entrypoints (history > synced tabs, onboarding tour, etc.).

You remind me that I should check this state from the other entrypoints.

**Comment by ryanfeeley** [comment]
For all other entrypoints, add a red Remove to the top left ![remove](https://user-images.githubusercontent.com/68213/68336126-bf78e400-00ab-11ea-8e5e-c7e35b4b4a90.png)

**Comment by athomasmoz** [comment]
@bryanbell are you good with the mocks Ryan added above?

**Comment by nbhasin2** [comment]
After talking with @bryanbell below are the new suggestions. 
- The "done" button on the presented modal should say "close"
- The "close" button should be on the left side and "remove" on the right for consistency 
- Whenever user presses "remove" we present them with an alert to show "Cancel" or "Remove" explaining the choices they have.

**Comment by nbhasin2** [comment]
FYI - Just waiting for finalizing the copy, until then moving to Needs UX

**Comment by bryanbell** [comment]
Can the copy for the dialog be...

```
Remove Account

Remove the Firefox Account associated with this device to sign in as a different user.

[ Cancel ] [ Remove ]
```

**Comment by SimonBasca** [comment]
Verifying as fix on master c2574a0a.
